### DISCUSSION TITLE
Can we read queue data created by bullmq from python consumer

### DISCUSSION THREAD

**Discussion by muhammad-omer-dev** [taskforcesh/bullmq#1648] [Q&A]
Hi,
I am currently running application in node.js and need to write python code for OCR services but for communication i am using Bullmq right now. I need to Publish data from Bullmq into queue and consume that data using Python Is this possible ?. Or any workaround for it ? Really need help on this.

**Comment by manast** [comment]
When you say that you want to consume data, do you mean processing jobs from the queue or something else?

**Comment by roggervalf** [comment]
Python versión is available https://docs.bullmq.io/python/introduction
